frontend/src/scenes/organization/Settings/VerifiedDomains/VerifiedDomains.tsx

```diff
@@ -11,9 +11,12 @@ import { InfoCircleOutlined, PlusOutlined } from '@ant-design/icons'
 import { LemonButton } from 'lib/components/LemonButton'
 import { More } from 'lib/components/LemonButton/More'
 import { AddDomainModal } from './AddDomainModal'
-import { SSOSelect } from './SSOSelect'
+//import { SSOSelect } from './SSOSelect'
```

Comment:
Commented out stuff relates to a temporary hiding of the SSO enforcement column.

Reply:
Normally I'm against committing in commented out code. I'm sure you'll follow up quickly and remove it, but what about just hiding it behind a boolean? The code has to do with `isSSOEnforcementAvailable` being available, so just force it to `false` and don't show the data?

Reply:
Not exactly that but you have a valid point, even if it's a very short-lived thing better not to have it, and actually come to think of it, even for the real version we won't want the column to show in self-hosted instances, so doing that instead.